docker/nautilus.sh

```diff
@@ -0,0 +1,257 @@
+
+
+    POSTGRES_WORK_DIR="$WORKING_DIR"/postgres-"$DEPLOYMENT_ENV"
+    mkdir "$POSTGRES_WORK_DIR"
+
+    cp "$DIR"/app/postgres/dockerfile "$POSTGRES_WORK_DIR"/dockerfile
+    postgres_dockerfile="$POSTGRES_WORK_DIR"/dockerfile
+    #change postgres databasename, username, and password
+    {
+    read line1
+    read line2
+    read line3
+    } < "$PATH_TO_CONFIG"/postgres/credentials.txt
+    line1=`echo $line1`
+    line2=`echo $line2`
+    line3=`echo $line3`
+
+    sed -i "s/ENV POSTGRES_USER=.*/$line1/g" "$postgres_dockerfile"
+    sed -i "s/ENV POSTGRES_PASSWORD=.*/$line2/g" "$postgres_dockerfile"
+    sed -i "s/ENV POSTGRES_DB=.*/$line3/g" "$postgres_dockerfile"
+
+
+
+    #check out schema and put it in the right place
+    mv "$PATH_TO_CONFIG"/postgres/conseil.sql "$PATH_TO_CONFIG"/postgres/conseil.sql.bak
+    wget https://raw.githubusercontent.com/Cryptonomic/Conseil/master/doc/conseil.sql > "$PATH_TO_CONFIG"/postgres/conseil.sql
+
+    #place schema into working directory for auditability
+    cd "$POSTGRES_WORK_DIR"
+    rm ./conseil.sql
+    cp "$PATH_TO_CONFIG"/postgres/conseil.sql ./conseil.sql
+    docker build -f dockerfile -t postgres-"$DEPLOYMENT_ENV" .
+
+	docker run --name=postgres-"$DEPLOYMENT_ENV" --network=nautilus -v pgdata-"$DEPLOYMENT_ENV":/var/lib/postgresql/data -d -p 5432:5432 postgres-"$DEPLOYMENT_ENV"
+}
+
+build_tezos () {
+    #volumes creation for persistent storage
+    [[ -d $HOME/volumes ]] || mkdir $HOME/volumes
+    [[ -d $HOME/volumes/tznode_data-"$DEPLOYMENT_ENV" ]] || mkdir $HOME/volumes/tznode_data-"$DEPLOYMENT_ENV"
+    [[ -d $HOME/volumes/tzclient_data-"$DEPLOYMENT_ENV" ]] || mkdir $HOME/volumes/tzclient_data-"$DEPLOYMENT_ENV"
+    #stop and remove current container
+    docker container stop tezos-node-"$DEPLOYMENT_ENV"
+	docker container rm tezos-node-"$DEPLOYMENT_ENV"
+
+    #createdocker volumes
+    docker volume create --driver local --opt type=none --opt o=bind --opt device=$HOME/volumes/tznode_data-"$DEPLOYMENT_ENV" tznode_data-"$DEPLOYMENT_ENV"
+    docker volume create --driver local --opt type=none --opt o=bind --opt device=$HOME/volumes/tzclient_data-"$DEPLOYMENT_ENV" tzclient_data-"$DEPLOYMENT_ENV"
+
+	#make tezos subdirectory
+    TEZOS_WORK_DIR="$WORKING_DIR"/tezos
+    mkdir "$TEZOS_WORK_DIR"
+
+    #copy dockerfile from nautilus
+    cp "$DIR"/app/tezos/dockerfile "$TEZOS_WORK_DIR"/dockerfile
+
+    #replace tezos network in dockerfile
+    tz_dockerfile="$TEZOS_WORK_DIR"/dockerfile
+    sed -i "s/protocol/$tezosnetwork/g" "$tz_dockerfile"
+    cd "$TEZOS_WORK_DIR"
+
+    #build and run docker container
+    docker build -f dockerfile -t tezos-node-"$DEPLOYMENT_ENV" .
+    docker run --name=tezos-node-"$DEPLOYMENT_ENV" --network=nautilus -v tznode_data:/var/run/tezos/node-"$DEPLOYMENT_ENV" -v tzclient_data:/var/run/tezos/client-"$DEPLOYMENT_ENV" -d -p 8732:8732 -p 9732:9732 tezos-node-"$DEPLOYMENT_ENV"
+}
+
+remove_postgres_volumes () {
```

Comment:
Please rename this function to indicate that it actually fully removes Postgres, i.e. both the container and the volume.

Reply:
fixed, remove_postgres()